test/e2e/apps/job.go

```diff
@@ -48,6 +53,18 @@ import (
 	"github.com/onsi/gomega"
 )
 
+type watchEventConfig struct {
+	framework       *framework.Framework
+	resourceVersion string
+	w               *cache.ListWatch
+	jobName         string
+	watchEvent      watch.EventType
+	extJob          *batchv1.Job
+	internalKind    string
+	internalValue   string
+	externalValue   string
```

Comment:
these variable names are very confusing.

I would say:
```
updatedMetadataType
updatedKey
updatedValue
```

but I'm open to suggestions

Reply:
Fixed!. Thank for the suggested wording as I wasn't happy with it either but now we are! 💯